src/index.js

```diff
@@ -52,10 +52,14 @@ export default function testStorySnapshots (options = {}) {
   const stories = storybook.getStorybook()
 
   for (const group of stories) {
+    if (options.storyKindRegex && !group.kind.match(options.storyKindRegex)) {
+      continue
+    }
+
     describe(suit, () => {
       describe(group.kind, () => {
         for (const story of group.stories) {
-          if (options.storyRegex && !story.name.match(options.storyRegex)) {
+          if (options.storyNameRegex && !story.name.match(options.storyNameRegex)) {
```

Comment:
Also support `options.storyRegex` so we don't break any storybooks. It's okay to remove it from the Readme file.